modules/core/src/core/animation-loop.js

```diff
@@ -268,12 +380,16 @@ export default class AnimationLoop {
         this._nextFramePromise = null;
         this._resolveNextFrame = null;
       }
-      this._animationFrameId = requestAnimationFrame(renderFrame);
+      this._animationFrameId = requestAnimationFrame(renderFrame, this._animationFrameDevice());
     };
 
     // cancel any pending renders to ensure only one loop can ever run
     cancelAnimationFrame(this._animationFrameId);
-    this._animationFrameId = requestAnimationFrame(renderFrame);
+    this._animationFrameId = requestAnimationFrame(renderFrame, this._animationFrameDevice());
+  }
+
+  _animationFrameDevice() {
+    return this.vrPresenting ? this.vrDisplay : window;
```

Comment:
Beware: Take care to avoid referencing window in case we are running under Node.js

Reply:
hmmmm....